clients/naming_client/naming_cache/cache/naming/DEFAULT_GROUP@@bug_267.com

```diff
@@ -0,0 +1 @@
+{"dom":"","cacheMillis":10000,"useSpecifiedUrl":false,"hosts":[{"valid":false,"marked":false,"instanceId":"10.10.10.100#3600#DEFAULT#DEFAULT_GROUP@@bug_267.com","port":3600,"ip":"10.10.10.100","weight":50,"metadata":{},"clusterName":"DEFAULT","serviceName":"DEFAULT_GROUP@@bug_267.com","enabled":true,"healthy":true,"ephemeral":false}],"checksum":"","lastRefTime":1622688135226,"env":"","clusters":"","metadata":null,"name":"DEFAULT_GROUP@@bug_267.com"}
```

Comment:
unnecessary

Reply:
If don`t add this, the test case can`t run.